ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
Ok, the implementation for these is super simple, so the change would be easy. Just need to add back compat.